addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1510,7 +1510,8 @@ export function isHtmlContentSupported(node) {
  */
 const selfClosingElementTags = ['BR', 'IMG', 'INPUT'];
 export function isSelfClosingElement(node) {
-    return node && selfClosingElementTags.includes(node.nodeName);
+    return node && (selfClosingElementTags.includes(node.nodeName) ||
+        (node.nodeType === Node.ELEMENT_NODE && node.attributes.getNamedItem('role')?.value === 'img'));
```

Comment:
I'm not sure I get it. I don't think we should remove font awesome nodes and whether they have role="img" is irrelevant for that matter IMO. Did I miss something ?

Reply:
@dmo-odoo Do you mean that I should specifically add the `I` tag to the `selfClosingElementTags` and not consider the `role="img"` attribute ?